client/scss/components/_modals.scss

```
@@ -116,7 +124,10 @@ $zindex-modal-background: 500;
 
 @include media-breakpoint-up(sm) {
   .modal-dialog {
-    padding: 0 0 2em $menu-width;
+    width: auto;
+    padding: 0 0 2em 0;
+    margin-inline-start: ($menu-width) * 1.5;
+    margin-inline-end: 6em;
```

when adding margin start/end - it would be best to use the same values either side here, plus adopt theme spacing.

```suggestion
    margin-inline-end: theme('spacing.20');
    margin-inline-start: theme('spacing.20');
```

Also, when adding margins to the sm -> md breakpoint, you will need to ensure the margin auto is added for the larger breakpoints.